tests/src/TestSuite.hx

```diff
@@ -23,15 +23,15 @@ class TestSuite extends massive.munit.TestSuite
 	{
 		super();
 
+		add(haxepunk.EntityListTest);
```

Comment:
munit sorts the test names so they don't change too often...but kind of defeats the purpose if our systems sort them differently :)